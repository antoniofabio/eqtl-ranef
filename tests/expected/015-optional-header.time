	Command being timed: "./test-015-optional-header.sh"
	User time (seconds): 5.34
	System time (seconds): 0.09
	Percent of CPU this job got: 100%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:05.44
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 513664
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 36615
	Voluntary context switches: 19
	Involuntary context switches: 709
	Swaps: 0
	File system inputs: 0
	File system outputs: 16
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
