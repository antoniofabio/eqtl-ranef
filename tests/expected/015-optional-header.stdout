reporterID	regressorID	pvalue	beta	t	nobs	isCis
1007_s_at	rs1	1.67093397348574e-36	-4.04857761680287	-12.6184424628884	7	cis
1053_at	rs1	0.798637480077157	1.36512385563221	0.255110855439918	7	trans
1007_s_at	rs2	0.341469612263285	1.61617566003394	0.951265506754245	7	trans
1053_at	rs2	0.376907726409755	-2.41735264536876	-0.883608245490484	7	cis
